### PR TITLE
Import cluster api example doc

### PR DESCRIPTION
**Pull request by anivargi**

### CODE REVIEW

**Comment by anivargi** [comment]
@brainfunked @nthomas-redhat please review.

**Review by brainfunked — APPROVED** [review]

File: api/import-cluster-example.adoc [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Sample Response
+
+----------
+Status: 200 OK
+
+[
+  {
+    "node_id":"7026f371-2d8c-4ad7-895f-4d4dcc70ff1c",
+    "fqdn":"dhcp41-239.lab.tendrl.org",
```
Comment (shtripat) [inline api/import-cluster-example.adoc:94]: Better dont put the absolute node names :)

File: api/import-cluster-example.adoc [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+----------
+Status: 202 Accepted
+
+{ job_id: "3784922e33e8bec939be5e626e21a174" }
+----------
+
+The `job_id` can be used to poll the status of the job.
+The import cluster action queued a job in etcd which will be picked up by the
+node agent and processed.
+----
```
Comment (shtripat) [inline api/import-cluster-example.adoc:124]: Shouldnt we add a sample for fetching the job status as well here?

**Comment by anivargi** [comment]
Closing, does not need to be merged anymore, will be part of other PR

**Pull request closed without merging**